core/commands/repo.go

```diff
@@ -152,3 +158,49 @@ RepoPath        string the path to the repo being currently used
 		},
 	},
 }
+
+var repoFsckCmd = &cmds.Command{
+	Helptext: cmds.HelpText{
+		Tagline: "Removes repo lockfiles",
+		ShortDescription: `
+'ipfs repo fsck' is a plumbing command that will remove repo and level db lockfiles
+`,
+	},
+	Run: func(req cmds.Request, res cmds.Response) {
+
+		configRoot := req.InvocContext().ConfigRoot
+
+		// acquire the repo lock before removing it
+		repo, err := fsrepo.Open(configRoot)
```

Comment:
I don't know if this will work correctly... i think the reason we want to remove the lockfiles is because theyre in a bad state and we *cant* take the locks.

We should however try and check if the daemon is online.